tests/test_wheelfile_utils.py

```diff
@@ -24,7 +24,8 @@ def test_wheel_writer_simple(tmp_path, monkeypatch):
                 "name": "something",
                 "version": "1.2.3",
             },
-        }
+        },
+        metadata_version="2.3",
```

Comment:
Why is this 2.3 then?

Reply:
Well, it's explicitly testing that the manual override works, e.g. when manually setting it to 2.4 for the recent PEP639 support.

Reply:
Yeah, I just picked a random one here since this unit test doesn't include our manual "if 2.1" check. The one thing I didn't add is a check to see if it it still 2.1 if the minimum version is set below 0.11. If this had included our override, I'd have done that here.

Reply:
Also, related, I'm really thinking it might be safer to drop 3.7 support in 0.11 to go along with the default change, so 3.7 users don't have to worry about support for METADATA 2.2.

Reply:
I'll likely bring this up in today's developer meeting. :)